src/components/clients/ClientForm.js

```diff
@@ -18,6 +27,28 @@ class ClientForm extends React.Component {
     });
   }
 
+  onRedirectUrlChange(e) {
+    const focus = this.state[e.target.name + 'Focus'];
+    let urls = Object.assign([], this.state[e.target.name]);
+    urls[focus] = e.target.value;
+    this.setState({
+      [e.target.name]: urls
+    });
+  }
+
+  onClick(e) {
+    e.preventDefault();
```

Comment:
note: if you use `<button type="button">` it doesn't submit the form when you click it, and so you don't need preventDefault. I think it's also a good accessibility practice.